GCSViews/ConfigurationView/ConfigRawParams.cs

```diff
@@ -318,11 +359,14 @@ private void BUT_writePIDS_Click(object sender, EventArgs e)
                     CustomMessageBox.Show("Set " + value + " Failed");
                 }
             }
+            

```

Comment:
Drop this too.

Reply:
dropped in [commit id 22c75ed](https://github.com/ArduPilot/MissionPlanner/pull/3412/commits/22c75ed012c5702bd9eb70df164d8166320fa3e3)